.ci/ci.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
 - job: Linux
   timeoutInMinutes: 360
   pool:
-    vmImage: ubuntu-16.04
+    vmImage: ubuntu-latest
```

Comment:
Using `ubuntu-latest` sounds reasonable to me.

Reply:
Nice, also this is the same for mlpack too, so should i open a PR there too ?

Reply:
For mlpack we wanted to make sure we run tests against the oldest version, so we will wait until September 20.

Reply:
Sure, maybe we can leave it that way.